src/plugins/justwatch/hubs/JustWatchHub.ts

```diff
@@ -0,0 +1,109 @@
+import * as plexTypes from '../../../plex/types';
+import { PseuplexRequestContext } from '../../../pseuplex';
+import { JustWatchTitle, JustWatchHubConfig } from '../types';
+import { JustWatchMetadataProvider } from '../metadata';
+import { getPopularTitles } from '../api';
+import { APIFeedHub, APIFeedHubOptions } from './APIFeedHub';
+
+export type JustWatchHubOptions = Omit<APIFeedHubOptions<JustWatchTitle>, 'apiCall' | 'dataTransformer'> & {
+	config: JustWatchHubConfig;
+	justWatchMetadataProvider?: JustWatchMetadataProvider;
+};
+
+/**
+ * JustWatch-specific hub that uses the generic APIFeedHub
+ * with JustWatch API calls and transformations
+ */
+export class JustWatchHub extends APIFeedHub<JustWatchTitle> {
+	private readonly justWatchConfig: JustWatchHubConfig;
+	private readonly justWatchMetadataProvider?: JustWatchMetadataProvider;
+
+	constructor(options: JustWatchHubOptions) {
```

Comment:
Stuck with the conventions [here](https://github.com/lufinkey/pseuplex/blob/dev/src/plugins/letterboxd/hubs.ts) and make methods to create a specific hubs using your generalized hub class